olive/passes/olive_pass.py

```diff
@@ -463,12 +464,30 @@ def _run_for_config(
 
 
 # TODO(jambayk): rename. We are using FullPassConfig since PassConfigBase already refers to inner config
-class FullPassConfig(ConfigBase):
-    type: str
-    disable_search: bool = False
+class CommonPassConfig(ConfigBase):
```

Comment:
nit: `Common` seems a misnomer here. Use 'Base` or 'Abstract' instead.

Reply:
done! change base to abstract.